Makefile

```diff
@@ -54,16 +54,17 @@ endif
 # Version-specific settings
 ifeq ($(VERSION),gc-us)
   DEBUG := 0
-  CPP_DEFINES += -DTEXT_LANGUAGE=TEXT_LANG_US_JP
+  COMPARE := 0
+  CPP_DEFINES += -DOOT_NTSC
 else ifeq ($(VERSION),gc-eu)
   DEBUG := 0
-  CPP_DEFINES += -DTEXT_LANGUAGE=TEXT_LANG_EU
+  CPP_DEFINES += -DOOT_PAL
 else ifeq ($(VERSION),gc-eu-mq)
   DEBUG := 0
-  CPP_DEFINES += -DTEXT_LANGUAGE=TEXT_LANG_EU -DOOT_MQ
+  CPP_DEFINES += -DOOT_PAL -DOOT_MQ
```

Comment:
btw I would prefer we explicitly set the macros to 0 or 1 rather than relying on the value of a macro that was merely defined be 1, and expansion of a macro being undefined be 0
can do later though

Reply:
I can go ahead and do this now, should we keep `OOT_PAL` or use `!OOT_NTSC` everywhere instead?

Reply:
Personally I much better like `OOT_PAL` and `OOT_NTSC` rather than doing mental gymnastics, and it seems I'm not alone since we introduced `OOT_DEBUG=0/1` in that spirit of avoiding mental gymnastics